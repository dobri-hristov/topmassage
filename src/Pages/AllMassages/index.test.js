import React from 'react'
import TestRenderer from 'react-test-renderer'
import MassagesPage from './index.js'

describe('Massages page', () => {
    test('should render correctly with no data', () => {
        const component = TestRenderer.create(<MassagesPage />)
        const tree = component.toJSON()
        expect(tree).toMatchSnapshot()
    })
})