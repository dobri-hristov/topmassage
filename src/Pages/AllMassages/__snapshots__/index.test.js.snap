// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Massages page should render correctly with no data 1`] = `
<div>
  <div
    className="container black"
  >
    <h2>
      Масажи
    </h2>
  </div>
  <div
    style={
      Object {
        "textAlign": "center",
      }
    }
  >
    <span
      className="MuiCircularProgress-root MuiCircularProgress-indeterminate MuiCircularProgress-colorPrimary css-18lrjg1-MuiCircularProgress-root"
      role="progressbar"
      style={
        Object {
          "height": "8rem",
          "width": "8rem",
        }
      }
    >
      <svg
        className="MuiCircularProgress-svg css-1idz92c-MuiCircularProgress-svg"
        viewBox="22 22 44 44"
      >
        <circle
          className="MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate css-176wh8e-MuiCircularProgress-circle"
          cx={44}
          cy={44}
          fill="none"
          r={20.2}
          strokeWidth={3.6}
          style={Object {}}
        />
      </svg>
    </span>
  </div>
</div>
`;
